### PR TITLE
docs(README): adding MacOS to release process topic

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ What makes KICS really powerful and popular is its built-in extensibility. This
 ## Release process
 
 KICS release process is quite simple. We have nightly builds that will pack and pre-release all changes merged into master.
-The nightly release will have a "nightly" prefix with the last commit hash code. We have binaries available for both Windows and Linux, as well a Docker image in <a href="https://hub.docker.com/r/checkmarx/kics" target="_blank">DockerHub</a>
+The nightly release will have a "nightly" prefix with the last commit hash code. We have binaries available for both Windows, Linux and MacOS, as well a Docker image in <a href="https://hub.docker.com/r/checkmarx/kics" target="_blank">DockerHub</a>
```
Comment (felipe-avelar) [inline README.md:62]:
```suggestion
The nightly release will have a "nightly" prefix with the last commit hash code. We have binaries available for Windows, Linux and MacOS, as well a Docker image in <a href="https://hub.docker.com/r/checkmarx/kics" target="_blank">DockerHub</a>
```

**Review by felipe-avelar — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ What makes KICS really powerful and popular is its built-in extensibility. This
 ## Release process
 
 KICS release process is quite simple. We have nightly builds that will pack and pre-release all changes merged into master.
-The nightly release will have a "nightly" prefix with the last commit hash code. We have binaries available for both Windows and Linux, as well a Docker image in <a href="https://hub.docker.com/r/checkmarx/kics" target="_blank">DockerHub</a>
+The nightly release will have a "nightly" prefix with the last commit hash code. We have binaries available for Windows, Linux and MacOS, as well a Docker image in <a href="https://hub.docker.com/r/checkmarx/kics" target="_blank">DockerHub</a>
```
Comment (nunoocx) [inline README.md:62]: ... as well AS a Docker...

Comment (rogeriopeixotocx) [reply README.md]: solved!

**Review by nunoocx — APPROVED** [review]
LGMT

**Pull request merged**